admin/resources/js/components/classification/UpdateClassification.tsx

```diff
@@ -74,12 +75,21 @@ export const UpdateClassificationForm: React.FunctionComponent<UpdateClassificat
               type="text"
               rules={{ required: errorMessages.required }}
             />
-            <Input
+            <Select
               id="level"
               name="level"
               label="Level: "
-              type="number"
-              rules={{ required: errorMessages.required }}
+              options={[
```

Comment:
I would add this to the array of options so that the user has to choose the value so they don't overlook the field:
`{ value: "", label: "Select a level...", disabled: true },`
You'll also want to add a translation string for _Select a level..._